### PR TITLE
cleaned up some templating

### PR DESCRIPTION
**Pull request by mywyau**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -293,3 +293,11 @@ confirmation.refNumber = Your reference number
 confirmation.p1 = HMRC has successfully received your return
 confirmation.whatDoYouThink.link = What did you think of this service?
 confirmation.whatDoYouThink.time = (takes 30 seconds)
+
+
+# mikeyPage Messages
```
Comment (yangwong9664) [inline conf/messages.en]: mikey page

File: conf/messages.en [hunk 2]
```diff
@@ -293,3 +293,11 @@ confirmation.refNumber = Your reference number
...
+# ----------------------------------------------------------
+mikey.title = mikey
```
Comment (yangwong9664) [inline conf/messages.en]: mikey.mikey

File: conf/messages.cy [hunk 3]
```diff
@@ -279,3 +279,11 @@ confirmation.refNumber = Your reference number
...
 confirmation.whatDoYouThink.link = What did you think of this service?
 confirmation.whatDoYouThink.time = (takes 30 seconds)
+
+
+# mikeyPage Messages
+# ----------------------------------------------------------
+mikey.title = mikey
+mikey.heading = yooooo
+mikey.checkYourAnswersLabel = yooooo
+mikey.error.required = Select yes if yooooo
```
Comment (yangwong9664) [inline conf/messages.cy]: souja boi YOOOOOO

File: .g8/page/generated-test/views/$className$ViewSpec.scala [hunk 4]
```diff
@@ -8,16 +8,12 @@ class $className$ViewSpec extends ViewBehaviours {
...
-    case (html, templatingSystem) =>
-      s"$className$View (\$templatingSystem)" must {
+      s"$className$View" must {
 
-        behave like normalPage(html, "$className;format="
-        decap"$"
+        behave like normalPage(twirlView, "$className;format="decap"$"
         )
-        behave like pageWithBackLink(html)
+        behave like pageWithBackLink(twirlView)
```
Comment (yangwong9664) [inline .g8/page/generated-test/views/$className$ViewSpec.scala]: just view?

File: conf/messages.cy [hunk 5]
```diff
@@ -287,4 +287,4 @@ confirmation.whatDoYouThink.time = (takes 30 seconds)
 deemedParent.title = Does the group have a 'deemed' parent company?
 deemedParent.heading = Does the group have a 'deemed' parent company?
 deemedParent.checkYourAnswersLabel = Does the group have a 'deemed' parent company?
-deemedParent.error.required = Select yes if the group has a ’deemed’ parent company?
\ No newline at end of file
+deemedParent.error.required = Select yes if the group has a 'deemed' parent company?
```
Comment (yangwong9664) [inline conf/messages.cy]: 'deemed' should be ‘deemed’

File: conf/messages.cy [hunk 6]
```diff
@@ -287,4 +287,4 @@ confirmation.whatDoYouThink.time = (takes 30 seconds)
 deemedParent.title = Does the group have a 'deemed' parent company?
 deemedParent.heading = Does the group have a 'deemed' parent company?
 deemedParent.checkYourAnswersLabel = Does the group have a 'deemed' parent company?
```
Comment (yangwong9664) [inline conf/messages.cy]: 'deemed' should be ‘deemed’

File: conf/messages.cy [hunk 7]
```diff
@@ -287,4 +287,4 @@ confirmation.whatDoYouThink.time = (takes 30 seconds)
 deemedParent.title = Does the group have a 'deemed' parent company?
 deemedParent.heading = Does the group have a 'deemed' parent company?
```
Comment (yangwong9664) [inline conf/messages.cy]: 'deemed' should be ‘deemed’

File: conf/messages.cy [hunk 8]
```diff
@@ -287,4 +287,4 @@ confirmation.whatDoYouThink.time = (takes 30 seconds)
 deemedParent.title = Does the group have a 'deemed' parent company?
```
Comment (yangwong9664) [inline conf/messages.cy]: 'deemed' should be ‘deemed’

File: .g8/checkboxPage/migrations/$className__snake$.sh [hunk 9]
```diff
@@ -6,45 +6,43 @@ echo "Applying migration $className;format="snake"$"
 echo "Adding routes to conf/app.routes"
 
 echo "" >> ../conf/app.routes
```
Comment (yangwong9664) [inline .g8/checkboxPage/migrations/$className__snake$.sh]: pls check, doesn't work

File: conf/messages.cy (hunk 8 above)
Comment (mywyau) [reply conf/messages.cy]: i need spare special apostrophes  :'(

**Pull request merged**